CHANGELOG.md

```diff
@@ -14,7 +14,7 @@
 - Added several Relay built-in directives.
 - Supported suppression of inspections with comment for the definition or the whole file.
 - Changing the file highlight level in the editor prevents any inspections from being displayed.
-- Load variables from .env files. Supported file names: `.env.local`,`.env.development.local`,`.env.development`,`.env`.
+- Load variables from .env files. Supported file names: `.env.local`,`.env.development.local`,`.env.development`,`.env.dev.local`,`.env.dev`,`.env`.
```

Comment:
Please, revert it. It's a changelog for the already released version.

Reply:
You could add a new entry for the next version 3.1.0 like 
```
## [3.0.0] - 2021-07-16

### Added
- <description of your change>
```

Reply:
done